config/rewrites.d/support.rackspace.com.json

```diff
@@ -931,7 +931,7 @@
         "rewrite": false,
         "status": 301
       },
-     {
+      {
         "description": "redirect supported OS article",
         "from": "^\\/how-to\\/supported-operating-systems(.*)$",
         "to": "/how-to/supported-operating-systems-in-rackspace-public-cloud/",
```

Comment:
@DonNateR Can you check the indentation for the rest of the block? I think L940 is off, at least, but I can't be 100% sure without pulling down the PR...

Reply:
never mind; checked in person